libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -363,6 +363,16 @@ class AP_AHRS
         return groundspeed_vector().length();
     }
 
+    // return ground course estimate in degrees from 0 to 360
+    float ground_course(void) {
+        return wrap_360(degrees(groundspeed_vector().angle()));
```

Comment:
what would be the difference between this and using the yaw_sensor like in other place ? (like in https://github.com/ArduPilot/ardupilot/blob/master/libraries/GCS_MAVLink/GCS_Common.cpp#L2631)